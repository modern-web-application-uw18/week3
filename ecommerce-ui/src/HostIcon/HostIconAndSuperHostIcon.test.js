import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import SuperHostIcon from './SuperHostIcon';
import HostIcon from './HostIcon';

describe('SuperHostIcon component', () => {
  it('should render', () => {
    const component = ReactTestRenderer.create(<SuperHostIcon />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});

describe('HostIcon component', () => {
  it('should render', () => {
    const component = ReactTestRenderer.create(<HostIcon />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
