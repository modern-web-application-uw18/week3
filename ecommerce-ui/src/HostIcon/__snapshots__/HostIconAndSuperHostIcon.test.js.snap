// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HostIcon component should render 1`] = `
<svg
  fill="none"
  height="24"
  stroke="rgb(130, 110, 85)"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"
  />
  <circle
    cx="12"
    cy="7"
    r="4"
  />
</svg>
`;

exports[`SuperHostIcon component should render 1`] = `
<svg
  fill="none"
  height="24"
  stroke="rgb(130, 110, 85)"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M16 21v-2a4 4 0 0 0-4-4H5a4 4 0 0 0-4 4v2"
  />
  <circle
    cx="8.5"
    cy="7"
    r="4"
  />
  <line
    x1="20"
    x2="20"
    y1="8"
    y2="14"
  />
  <line
    x1="23"
    x2="17"
    y1="11"
    y2="11"
  />
</svg>
`;
